tests/analyze_ecl_failure.sh

```diff
@@ -24,7 +24,7 @@ do
     for line in $alines
     do
       abs_new=`echo $line| awk -F ' ' '{print $5}'`
-      abs_new=`echo ${abs_new: : -1} | awk '{printf sprintf("%.16f", $1); }'`
+      abs_new=`echo ${abs_new} | sed -e 's/\.$//g' | awk '{printf sprintf("%.16f", $1); }'`
```

Comment:
you are both correct. it's not equivalent, but it's also achieving the wanted thing. 

in particular,
we are catching lines like 
'The absolute deviation is 0.725098. The tolerance limit is 0.02' and my lazy awk'ing above end ups with '0.725098.'

the following awk printf (to get rid of exponential form of the capture) will not print anything if the last dot is there.